### PR TITLE
Pia 4368 reference docs

### PR DESCRIPTION
**Pull request by zladzeyka**
- Removed guide and unused resources, kept only reference documentation

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
Looks good to me! 👍

File: CaptureSDK/GiniCaptureSDK/Documentation/Sections/Documentation.md [hunk 1]
```diff
@@ -1,21 +1,2 @@
...
-The Gini Capture SDK provides components for capturing, reviewing and analyzing photos of invoices and remittance slips.
-
-By integrating this SDK into your application you can allow your users to easily take a picture of a document, review it and getting analysis results from the Gini backend.
-
-We provide pre-defined screens that can be customized in a limited way. The screen and configuration design is based on our long-lasting experience with integration in customer apps.
-
-On *iPhone*, the Gini Capture SDK has been designed for portrait orientation.
-On *iPad* we support portrait and landscape orientations.
 
+# Gini Capture SDK Reference documentation
```
Comment (ValentinaIancu-Gini) [inline CaptureSDK/GiniCaptureSDK/Documentation/Sections/Documentation.md:2]: [clarification] This file will remain empty?

Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/Sections/Documentation.md:2]: yes. Jazzy needs one page as a starting point for `index.html`.

Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Documentation/Sections/Documentation.md:2]: Thanks for telling me!

**Review by ValentinaIancu-Gini — APPROVED** [review]

**Pull request merged**